docs/README.md

```diff
@@ -1,20 +1,300 @@
 # User Guide
 
+Duke is a personal chat bot that uses the CLI to manage daily tasks. It is a case insensitve CLI application. 
+
+## Quick Start
+
+### Computing Environment
+
+Duke GUI has been tested and possible to run on Windows and Linux given the following computing environment
+* openjdk 11.0.4
+* OpenJDK Runtime Environment Build 11.0.4
+
+For Linux, please enter `java --version` to check if the minimum Java requirement is met before running the GUI application.
+For Windows, please enter `java --version` to check if the minimum Java requirement is met before running the GUI application.
+
+#### Start Duke Helpbot
+
+* Download the Duke Helpbot from [release page](https://github.com/huiminlim/duke/releases/tag/A-Release).
+
+#### For Linux, 
+* Go to the download location folder. You will see the `duke-0.2.0.jar` file. 
+* Right click the `duke-0.2.0.jar` file and click on `properties`. 
+![Linux Properties](linux_rightclickpermission.png)
+* Check the `Allow executing file as program` box.
+![Linux Check](linux_checkpermission.png)
+* Double click the `duke-0.2.0.jar` file to execute the Duke Helpbot GUI. 
+
+Alternatively, a command can be issued in the command terminal: `java -jar duke-0.2.0.jar`.
+
+#### For Windows,
+* Go to the download location folder. You will see the `duke-0.2.0.jar` file. 
+* Double click the `duke-0.2.0.jar` file to execute the Duke Helpbot GUI. 
+
+Alternatively, a command can be issued in the command terminal: `java -jar duke-0.2.0.jar`.
+
+
 ## Features 
 
-### Feature 1 
-Description of feature.
+Duke allows you to track three types of task - To Do, Deadline and Events. In addition, there are also various utility functions to allow you to manage your task list. These functions include `list`, `delete`, `done`, `bye`, `help`, `find`.
+
+For all To Do, Deadline and Events, a task description is shown in chronological order in which they were added. The classification of the To Do, Deadline and Events are denoted by a task icon, as shown in table below.
```

Comment:
a thorough readme! however, the language could be clearer and less verbose in some parts. :) some minor typos e.g. line 64: "shown to the you".